_posts/2015-09-04-channel.html

```diff
@@ -2708,6 +2725,15 @@
 				<td>string</td>
 				<td>Video file size in bytes</td>
 			</tr>
+			<tr>
+				<td>caption_matches</td>
+				<td>array of matched captions</td>
+				<td>
+					This field is only available if q parameter is used and there is a match in the video’s caption in the result set.
```

Comment:
"This field is only available" -> "This field is only present"
"if q parameter is used" -> "if the "q" parameter is used."
"and there is a match in the video’s caption in the result set." -> ""
"The result set contains the relevant part of the caption and their start_sec and end_sec timestamps that match with the search query." ->
"The result set contains captions matching the search query along with their start and end time ("start_sec" and "end_sec" fields respectively)."
"(Max 100 matched captions)" -> "Only a maximum of 100 matching captions are returned." vagy esetleg "100 matching captions are returned at a maximum."